server/middleware/hardStopCheckMiddleware.ts

```diff
@@ -1,20 +1,20 @@
 import { RequestHandler } from 'express'
-import LicenceKind from '../enumeration/LicenceKind'
 import UserType from '../enumeration/userType'
 import { isInHardStopPeriod } from '../utils/utils'
+import logger from '../../logger'
 
 export default function hardStopCheckMiddleware(userType: UserType): RequestHandler {
   return async (req, res, next) => {
     const { licence } = res.locals
     // Prison should only be able to access initial appointment update pages in hard stop, and probation should only be able to access
     // outside of hard stop
-    if (licence.kind === LicenceKind.VARIATION) {
-      return res.redirect('/access-denied')
-    }
+
```

Comment:
Removed from here as this is now a piece of middleware that's shared with other routes, where pages that are `VARIATION`s should be accessible _after_ the hard stop period.
eg:
- Before hard stop, PP can access, CA can't
- During hard stop, PP can't access, CA can
- After hard stop (variation or otherwise), PP can access, CA can't